### PR TITLE
Use v1beta2 as hub version

### LINKED ISSUE #479: AuthConfig v1beta2 stored in etcd
Change the stored version of the AuthConfig custom resources in etcd to [v1beta2](https://github.com/Kuadrant/authorino/blob/main/install/crd/authorino.kuadrant.io_authconfigs.yaml#L2184).

### PR DESCRIPTION
**Pull request by alexsnaps**
Unsure whether we want to change the controller to use v1beta2 as part of this PR or a different one...

Partially fixes #479 - need to update the controller logic to use the new API

### CODE REVIEW

**Comment by alexsnaps** [comment]
@guicassolato wdyt: should I add the controller changes to this PR?

**Comment by guicassolato** [comment]
@alexsnaps, can you please run `make generate manifests`?

**Comment by alexsnaps** [comment]
> @alexsnaps, can you please run `make generate manifests`?
> 
> There are still a few RBAC manifests that should not be part of this change.

I think [this is still not doing](https://github.com/Kuadrant/authorino/pull/482/commits/ae5b4c7eb243b6c403b75c5c60a6784f4fa47a0d) what you expect... I'm unsure what you are referencing here but `make generate manifests` "merely" changed these 2 lines in deepcopy 🤔

File: install/rbac/role.yaml [hunk 1]
```diff
@@ -41,3 +121,69 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - operator.authorino.kuadrant.io
+  resources:
+  - authorinos
```
Comment (guicassolato) [inline install/rbac/role.yaml:127]: I'm pushing a commit. IDK how this ended up here. This particular permission belongs to Authorino Operator, not to Authorino.

**Review by guicassolato — APPROVED** [review]
Tested and working as expected. Thanks, @alexsnaps!

**Pull request merged**